sdk/ai/Azure.AI.Projects/README.md

```diff
@@ -0,0 +1,356 @@
+# Azure AI Projects client library for .NET
+Use the AI Projects client library to:
+
+* **Develop Agents using the Azure AI Agent Service**, leveraging an extensive ecosystem of models, tools, and capabilities from OpenAI, Microsoft, and other LLM providers. The Azure AI Agent Service enables the building of Agents for a wide range of generative AI use cases. The package is currently in preview.
+* **Enumerate connections** in your Azure AI Studio project and get connection properties.
+For example, get the inference endpoint URL and credentials associated with your Azure OpenAI connection.
+
+[Product documentation](https://aka.ms/azsdk/azure-ai-projects/product-doc)
+| [Samples][samples]
+| [API reference documentation](https://aka.ms/azsdk/azure-ai-projects/csharp/reference)
+| [Package (NuGet)](https://aka.ms/azsdk/azure-ai-projects/csharp/package)
+| [SDK source code](https://aka.ms/azsdk/azure-ai-projects/csharp/source)
+
+## Getting started
+
+### Prerequisites
+
+To use Azure AI Projects capabilities, you must have an [Azure subscription](https://azure.microsoft.com/free/dotnet/). This will allow you to create an Azure AI resource and get a connection URL.
+
+### Install the package
+
+Install the client library for .NET with [NuGet](https://www.nuget.org/ ):
+
+```dotnetcli
+dotnet add package Azure.AI.Projects --prerelease
+```
+
+### Authenticate the client
+
+A secure, keyless authentication approach is to use Microsoft Entra ID (formerly Azure Active Directory) via the [Azure Identity library][azure_identity]. To use this library, you need to install the [Azure.Identity package](https://www.nuget.org/packages/Azure.Identity):
+
+```dotnetcli
+dotnet add package Azure.Identity
+```
+
+## Key concepts
+
+### Create and authenticate the client
+
+To interact with Azure AI Projects, you’ll need to create an instance of `AIProjectClient`. Use the appropriate credential type from the Azure Identity library. For example, [DefaultAzureCredential][azure_identity_dac]:
+
+```C# Snippet:OverviewCreateClient
+var connectionString = Environment.GetEnvironmentVariable("AZURE_AI_CONNECTION_STRING");
+AIProjectClient projectClient = new AIProjectClient(connectionString, new DefaultAzureCredential());
+```
+
+Once `AIProjectClient` is created, you can call `GetXXXClient()` methods on this client to retrieve instances of specific sub-clients.
+
+## Examples
+
+### Agents
+Agents in the Azure AI Projects client library are designed to facilitate various interactions and operations within your AI projects. They serve as the core components that manage and execute tasks, leveraging different tools and resources to achieve specific goals. The following steps outline the typical sequence for interacting with agents:
+
+Create an `AgentClient`
+```C# Snippet:OverviewCreateAgentClient
+var connectionString = Environment.GetEnvironmentVariable("AZURE_AI_CONNECTION_STRING");
```

Comment:
Was using a credential with a connection string intentional?   It's an odd pattern, so I wasn't sure if just an endpoint was meant for the environment variable.

Reply:
I totally agree, it’s not the most conventional pattern but this is the pattern supported by the service, and it’s referred to as a connection string. It combines the `<Endpoint>`, `<Subscription>`, and `<ProjectName>` into one string.